tests/pytorch_pfn_extras_tests/test_handler.py

```diff
@@ -6,6 +6,13 @@
 import pytorch_pfn_extras as ppe
 
 
+def torch_testing_assert_close(*args, **kwargs):
+    if ppe.requires("1.10.0"):
```

Comment:
note: `torch.testing.assert_close` has been supported from torch 1.9.0, but comparing Tensor and Parameter is not allowed in 1.9.0. This behavior was fixed from 1.10.0.

```
AssertionError: Except for scalars, type equality is required, but got <class 'torch.Tensor'> and <class 'torch.nn.parameter.Parameter'> instead.
```